Doc/library/concurrent.futures.rst

```diff
@@ -227,6 +229,59 @@ ThreadPoolExecutor Example
                print('%r page is %d bytes' % (url, len(data)))
 
 
+InterpreterPoolExecutor
+-----------------------
+
+The :class:`InterpreterPoolExecutor` class is a :class:`ThreadPoolExecutor`
+subclass that uses a pool of isolated interpreters to execute calls
+asynchronously.  Each interpreter is isolated from the others and thus
+can side-step the :term:`Global Interpreter Lock <global interpreter lock>`,
+allowing the use of multiple cores.  Interpreters mostly can't share
+objects between them, which means that, in most cases, only picklable
+objects can be executed and returned.
```

Comment:
```suggestion
InterpreterPoolExecutor
-----------------------

The :class:`InterpreterPoolExecutor` class, a :class:`ThreadPoolExecutor`
subclass, uses a pool of isolated interpreters to execute calls
asynchronously.  Since each interpreter is isolated from the others,
side-stepping the :term:`Global Interpreter Lock <global interpreter lock>` ,
multiple cores can be used.  Since Interpreters do not share
objects between them, in most cases, only picklable
objects can be executed and returned.
```